### PR TITLE
Fixed compile issues with cordapp tutorial caused by API changes in master

### PR DESCRIPTION
**Pull request by Clintonio**

### CODE REVIEW

File: java-source/src/main/java/com/example/contract/IOUContract.java [hunk 1]
```diff
@@ -29,15 +30,15 @@
...
+    public void verify(LedgerTransaction tx) {
         final AuthenticatedObject<Commands.Create> command = requireSingleCommand(tx.getCommands(), Commands.Create.class);
         requireThat(require -> {
             // Generic constraints around the IOU transaction.
             require.using("No inputs should be consumed when issuing an IOU.",
                     tx.getInputs().isEmpty());
             require.using("Only one output state should be created.",
                     tx.getOutputs().size() == 1);
-            final IOUState out = (IOUState) tx.getOutputs().get(0);
+            final IOUState out = (IOUState) tx.getOutputs().get(0).component1();
```
Comment (adagys) [inline java-source/src/main/java/com/example/contract/IOUContract.java]: better make use of the new API: `tx.outputsOfType(IOUState.class).get(0)`

File: kotlin-source/src/main/kotlin/com/example/contract/IOUContract.kt [hunk 2]
```diff
@@ -21,13 +22,13 @@ open class IOUContract : Contract {
...
      */
-    override fun verify(tx: TransactionForContract) {
+    override fun verify(tx: LedgerTransaction) {
         val command = tx.commands.requireSingleCommand<Commands.Create>()
         requireThat {
             // Generic constraints around the IOU transaction.
             "No inputs should be consumed when issuing an IOU." using (tx.inputs.isEmpty())
             "Only one output state should be created." using (tx.outputs.size == 1)
-            val out = tx.outputs.single() as IOUState
+            val out = tx.outputs.single().data as IOUState
```
Comment (adagys) [inline kotlin-source/src/main/kotlin/com/example/contract/IOUContract.kt]: same here: `tx.outputsOfType<IOUState>().single()`

Comment (Clintonio) [inline kotlin-source/src/main/kotlin/com/example/contract/IOUContract.kt]: Fixed.

File: java-source/src/main/java/com/example/contract/IOUContract.java (hunk 1 above)
Comment (Clintonio) [inline java-source/src/main/java/com/example/contract/IOUContract.java]: Fixed.

**Comment by allenQBE** [comment]
There are further fixes that I believe need to be implemented for proper API usage, as the tutorial is now on release-M14. RPCOps API operation ```networkMapUpdates()``` has been removed and cannot be called, I believe it has been renamed  ```networkMapFeed()```? There also should be updates to reflect the removal of ```TransactionType``` as mentioned in the changelog for Corda. It seems that there are still some variables using the deprecated methods, which causes the build to fail. I have a feature branch I'd be happy to push with the fixes already implemented if you'd like!

**Comment by adagys** [comment]
Sure feel free! That would be helpful. I just merged a PR that fixes the `TransactionType` change though.

**Pull request merged**